local/bin/js/algolia-create-object-lookup.js

```diff
@@ -0,0 +1,63 @@
+const algoliasearch = require('algoliasearch');
```

Comment:
this script is not currently in use as i ran into some truncation issues bundling the resulting JSON data file.   i would like to revisit this later on after we migrate to use hugo/esbuild to compile JS in the docs however, so i've kept it in the PR.